src/sentry/notifications/notifications/active_release.py

```diff
@@ -0,0 +1,240 @@
+from __future__ import annotations
+
+from typing import Any, Mapping, MutableMapping, Optional, Sequence, TypedDict
+from urllib.parse import quote, urlencode
+
+from sentry import analytics, features
+from sentry.eventstore.models import Event
+from sentry.models import (
+    GroupOwner,
+    GroupOwnerType,
+    Release,
+    ReleaseActivity,
+    ReleaseCommit,
+    Team,
+    User,
+)
+from sentry.notifications.notifications.rules import AlertRuleNotification, logger
+from sentry.notifications.types import ActionTargetType, NotificationSettingTypes
+from sentry.notifications.utils import (
+    get_group_settings_link,
+    get_integration_link,
+    get_interface_list,
+    has_alert_integration,
+    has_integrations,
+)
+from sentry.notifications.utils.participants import get_owners
+from sentry.plugins.base import Notification
+from sentry.rules import EventState
+from sentry.types.integrations import EXTERNAL_PROVIDERS, ExternalProviders
+from sentry.types.releaseactivity import ReleaseActivityType
+from sentry.utils import metrics
+from sentry.utils.http import absolute_uri
+
+
+class ActiveReleaseIssueNotification(AlertRuleNotification):
+    message_builder = "ActiveReleaseIssueNotificationMessageBuilder"
+    metrics_key = "release_issue_alert"
+    notification_setting_type = NotificationSettingTypes.ACTIVE_RELEASE
+    template_path = "sentry/emails/release_alert"
+
+    def __init__(
+        self,
+        notification: Notification,
+        event_state: EventState,
+        target_type: ActionTargetType,
+        target_identifier: int | None = None,
+        last_release: Optional[Release] = None,
+    ) -> None:
+        from sentry.rules.conditions.active_release import ActiveReleaseEventCondition
+
+        super().__init__(notification, target_type, target_identifier)
+        self.last_release = (
+            last_release
+            if last_release
+            else ActiveReleaseEventCondition.latest_release(notification.event)
+        )
+        self.event_state = event_state
+
+    def get_notification_title(self, context: Mapping[str, Any] | None = None) -> str:
+        return "Active Release alert triggered"
+
+    def send(self) -> None:
```

Comment:
@barkbarkimashark do we need to override this method? We have sub-methods we could overwrite instead if that helps

Reply:
Which sub-methods would you suggest. I think overriding the method here kinda makes sense.

Reply:
@barkbarkimashark I guess this change has already been merged? I guess you are doing things like creating `ReleaseActivity` rows so maybe it's fine to do what you're doing.